script.js

```diff
@@ -81,7 +64,7 @@ function addGenerateCompareUrlButton() {
 		}
 		
 		firstSelectedCommitSHA = selectedCheckboxes[0].value;
-		secondSelectedCommitSHA = checkboxes[lastIndexOfSelectedCheckboxes + 1].value;
+		secondSelectedCommitSHA = selectedCheckboxes[1].value;
```

Comment:
This will cause a bug were the compare will NOT be to the second selected checkbox but rather to the one before.

This is how github compare works but since this is a UI extension I want to make that part more logical